tcmur_cmd_handler.c

```diff
@@ -206,8 +206,10 @@ static struct unmap_state *unmap_state_alloc(struct tcmu_device *dev,
 		goto out_free_state;
 	}
 
-	state->refcount = 0;
+	/* released by allocator when done submitting unmaps */
+	state->refcount = 1;
 	state->error = false;
+	state->status = TCMU_STS_ASYNC_HANDLED;
```

Comment:
I think you can change that to TCMU_STS_OK to match state->error. Then if we get an error, unmap_state_put will set error and status.

Reply:
I agree